pkgs/top-level/aliases.nix

```diff
@@ -726,6 +726,7 @@ mapAliases ({
   liblastfm = libsForQt5.liblastfm; # Added 2020-06-14
   libmongo-client = throw "'libmongo-client' has been removed, upstream gone"; # Added 2023-06-22
   libpulseaudio-vanilla = libpulseaudio; # Added 2022-04-20
+  libqtav = throw "libqtav has been removed as it was abandoned upstream"; # Added 2024-07-14
```

Comment:
this alias isn't doing anything, as `libqtav` wasn't at the top-level

Reply:
Right. Do we have any solution for making aliases in `libsForQt5`?

Reply:
I don't know of any such deprecation process, nor was I able to find any in the git history. It seems people just drop it.

Examples:
* 4e8ad00ae81e70f4e4db7dfc7e67d9c853f0d6b6
* 0bd5909ea7ae25fb5cd2667128d3c161abcbdb76
* ea88f8313ea31b0e76cac6a396dab416dc93b014

Reply:
Alright, I just dropped it.